crates/nix_rs/src/env.rs

```diff
@@ -29,15 +32,22 @@ impl NixEnv {
         use sysinfo::{DiskExt, SystemExt};
         tracing::info!("Detecting Nix environment");
         let os = OS::detect().await;
-        tokio::task::spawn_blocking(|| {
+        let output = Command::new("groups").output().await.unwrap();
```

Comment:
`unwrap` should generally not be used (it will crash the app). This function already returns a `Result`, so you want to marshall this error in a `NixEnvError`.

See how we do this for `current_user`.